easybuild/easyconfigs/s/shapeit/shapeit-v2.r837_GLIBCv2.12-linux_x86_64.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Binary"
+
+name = 'shapeit'
+version = '2.r837.GLIBCv2.12'
+versionsuffix = '_linux_x86_64'
+
+homepage = 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'
+description = """ SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.  """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/']
+sources = ['shapeit.v%(version)s.Linux.static.tgz']
+
+# Following commands are used to untar the file and copy the content to install directory
+install_cmd = "tar xzvf shapeit.v%(version)s.Linux.static.tgz && HOME=%(builddir)s && "
+install_cmd += "cp -a %(builddir)s/* %(installdir)s"
```

Comment:
@rtripath89 this is exactly what the `Tarball` easyblock does, why not use that?

Also, why is the `HOME=%(builddir)s` part needed at all?